testsuite/MDAnalysisTests/analysis/test_diffusionmap.py

```diff
@@ -20,57 +20,70 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 from __future__ import print_function, absolute_import
-from unittest import TestCase
-import numpy as np
+
 import MDAnalysis
 import MDAnalysis.analysis.diffusionmap as diffusionmap
-from numpy.testing import (assert_almost_equal, assert_equal,
-                           assert_array_almost_equal,raises)
+import numpy as np
+import pytest
+from MDAnalysisTests.datafiles import PDB, XTC
+from numpy.testing import assert_array_almost_equal
 
 
-from MDAnalysisTests.datafiles import PDB, XTC
+class TestDiffusionmap(object):
+    @staticmethod
+    @pytest.fixture()
+    def u():
+        return MDAnalysis.Universe(PDB, XTC)
+
+    @staticmethod
+    @pytest.fixture()
+    def dist(u):
+        return diffusionmap.DistanceMatrix(u, select='backbone')
+
+    @staticmethod
+    @pytest.fixture()
+    def dmap(dist):
+        d_map = diffusionmap.DiffusionMap(dist)
+        d_map.run()
+        return d_map
 
+    @staticmethod
+    @pytest.fixture()
+    def eigvals(dmap):
+        return dmap.eigenvalues
```

Comment:
What should I do with it then? Since there is no class, I can't attach it with `self` in some other fixture. Should I directly use the values from the `dmap` fixture?